src/test/rmc-dependency-test/dependency3/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+pub fn get_int() -> i32 {
+    // Use a constant to force an MIR GlobalAllocation::Memory
+    // Use a non-i32 so there will be a conflict between this
+    // version and the other version.
+    let one = &(1 as i8);
```

Comment:
It also matters that this is 1 and not 0, right?

Reply:
Added comment